packages/dashboard/package.json

```diff
@@ -11,7 +11,7 @@
     "build": "build -- snowpack build"
   },
   "dependencies": {
-    "@dimensiondev/holoflows-kit": "=0.8.0-20210429202414-74fb9a1",
+    "@dimensiondev/holoflows-kit": "^0.9.0-20210726055325-dfbe7fd",
```

Comment:
```suggestion
    "@dimensiondev/holoflows-kit": "=0.9.0-20210726055325-dfbe7fd",
```

Reply:
move to workspace root?

Reply:
No, it contains some web-extension only thing